cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -903,422 +903,978 @@ frappe.pages['payroll-configuratio'].on_page_load = function(wrapper) {
+								custom_field = [
+									{
+										doctype: "Salary Structure Assignment",
+										label: rowData.salary_component,
+										field_type: "Check",
+										field_name: rowData.salary_component.toLowerCase().replace(/[^a-z]/g, '')+ rowData.abbr.toLowerCase().replace(/[^a-z]/g, ''),
+										depends: "",
+										insert_after: "custom_allowances"
+									},
+									{
+										doctype: "Salary Structure Assignment",
+										label: rowData.salary_component + " Value",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)